choreolib/src/main/native/cpp/choreo/lib/ChoreoTrajectoryState.cpp

```diff
@@ -49,14 +54,24 @@ frc::ChassisSpeeds ChoreoTrajectoryState::GetChassisSpeeds() const {
 ChoreoTrajectoryState ChoreoTrajectoryState::Interpolate(
     const ChoreoTrajectoryState& endValue, double i) const {
   frc::Pose2d interpPose = frc::Interpolate(GetPose(), endValue.GetPose(), i);
+
+  // TODO: Can this be done cleaner?
+  std::array<units::newton_t, 4> lerpFX, lerpFY;
+  for (int idx = 0; idx < 4; ++idx) {
+    lerpFX[idx] = wpi::Lerp(moduleForcesX[idx], endValue.moduleForcesX[idx], i);
+    lerpFY[idx] = wpi::Lerp(moduleForcesY[idx], endValue.moduleForcesY[idx], i);
+  }
```

Comment:
If we store these using an interpolatable type it could be cleaner.

Reply:
They're already inherently interpolatable. Locality of behavior is more important than abstraction.